src/System.Private.CoreLib/src/System/AppDomain.cs

```diff
@@ -461,78 +410,11 @@ private static RuntimeAssembly GetRuntimeAssembly(Assembly asm)
         [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
         private static extern void nSetNativeDllSearchDirectories(string paths);
 
-        private void SetupFusionStore(AppDomainSetup info, AppDomainSetup oldInfo)
-        {
-            Debug.Assert(info != null);
-
-            if (info.ApplicationBase == null)
-            {
-                info.SetupDefaults(RuntimeEnvironment.GetModuleFileName(), imageLocationAlreadyNormalized: true);
-            }
-
-            nCreateContext();
-
-            // This must be the last action taken
-            _FusionStore = info;
-        }
-
-        // Used to switch into other AppDomain and call SetupRemoteDomain.
-        //   We cannot simply call through the proxy, because if there
-        //   are any remoting sinks registered, they can add non-mscorlib
-        //   objects to the message (causing an assembly load exception when
-        //   we try to deserialize it on the other side)
-        private static object PrepareDataForSetup(string friendlyName,
-                                                        AppDomainSetup setup,
-                                                        string[] propertyNames,
-                                                        string[] propertyValues)
-        {
-            var newSetup = new AppDomainSetup(setup);
-
-            // Remove the special AppDomainCompatSwitch entries from the set of name value pairs
-            // And add them to the AppDomainSetup
-            //
-            // This is only supported on CoreCLR through ICLRRuntimeHost2.CreateAppDomainWithManager
-            // Desktop code should use System.AppDomain.CreateDomain() or
-            // System.AppDomainManager.CreateDomain() and add the flags to the AppDomainSetup
-            var compatList = new List<string>();
-
-            if (propertyNames != null && propertyValues != null)
-            {
-                for (int i = 0; i < propertyNames.Length; i++)
-                {
-                    if (string.Equals(propertyNames[i], "AppDomainCompatSwitch", StringComparison.OrdinalIgnoreCase))
```

Comment:
This kind of appcompat switches is not used in CoreCLR anymore.

Reply:
We document this [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting). Not sure if we can actually remove this functionality.

Reply:
This switch does nothing for a few years now. I am not sure whether we have ever shipped .NET Core with this switch doing anything. That documentation is in need of updating ... we certainly do not maintain Phone or Silverlight quirks that it is talking about.

Reply:
OK - makes sense